lib/common/utils.js

```diff
@@ -13,6 +13,18 @@ export const waitFor = function waitFor(ms) {
   return new Promise(resolve => setTimeout(resolve, ms || 0))
 }
 
+export const isString = function isString(obj) {
+  return typeof obj === 'string' || obj instanceof String
+}
+
+export const unique = array => Array.from(new Set(array))
```

Comment:
For `lodash` uniq, it has some internal refator actually, if array is big enough, it will use set and array. https://github.com/lodash/lodash/blob/master/.internal/baseUniq.js#L34-L38

Reply:
I see. Will revert it

Reply:
Thank you 😸